app/ledger.js

```diff
@@ -441,9 +448,24 @@ var initialize = (onoff) => {
         }
 
         getStateInfo(state)
+
         try {
-          client = ledgerClient(state.personaId,
-                                              underscore.extend(state.options, { roundtrip: roundtrip }, clientOptions), state)
+          client = ledgerClient(
```

Comment:
why did the indentation change?

i agree that line 446 is indented too much, that must have been a slip of the finger. i thought it would be:

          client = ledgerClient(state.personaId,
                                underscore.extend(state.options, { roundtrip: roundtrip }, clientOptions), state)